spec/requests/application_controller_spec.rb

```diff
@@ -0,0 +1,51 @@
+RSpec.describe ApplicationController, type: :request do
```

Comment:
I reconsidered the usefulness of these specs.
I think we can remove these. We are just (re)testing the mapping in the rails actionpack/lib/action_dispatch/middleware/exception_wrapper.rb  is correct.

The useful tests are the `spec/requests/errors_controller_spec.rb`,
since the tests the routes and the rendered views.